### ISSUE TITLE
Runit Service

### ISSUE THREAD

**Issue by TheCatster** [Kharacternyk/pacwall#47]
Would it be possible for you to include a runit service for this? If not, I'd be happy to PR one.

Thanks!

**Comment by Kharacternyk** [comment]
A PR is welcome. I'm curious how you combine pacman and runit. Do you use Artix?

**Comment by TheCatster** [comment]
Yeah, I use Artix and Void, but my workstation is currently on Artix. I'm honestly not too sure, I've never tried runit with pacman, but thought I would give it a shot. I presume since it's run at a system level I can just run `pacman` fine.

**Comment by Kharacternyk** [comment]
By combining them I meant having them both installed. Note that the systemd units are not systemwide but "userwide". Does runit have facilities to run services on behalf of user? Running `pacwall` on behalf of root would be very problematic.

**Comment by TheCatster** [comment]
Yes, they do. Slightly more complicated to set up, but I actually just did one for the emacs daemon. 
Check this out: [runit user](http://smarden.org/runit/faq.html#userservices)

**Comment by TheCatster** [comment]
Alright, so I swapped back to Arch on my main workstation for a few reasons, but still have Artix on my laptop. Should have some time in these next few days to finish up my service and do a PR.
